access/ms-teams/bot.go

```diff
@@ -33,10 +44,10 @@ type Bot struct {
 	graphClient *msapi.GraphClient
 	// botClient represents MS Bot Framework client
 	botClient *msapi.BotFrameworkClient
-	// mu users access mutex
+	// mu recipients access mutex
 	mu *sync.RWMutex
 	// apps represents the cache of apps
```

Comment:
Update comment to match code?
```suggestion
	// recipients represents the cache of potential message recipients
```